bucket/onboard-memory-manager.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "1.1.6805",
+    "description": "A utility for pro gamers to quickly view, customize, and fine-tune the onboard memory of a compatible Logitech G mouse",
+    "homepage": "https://support.logi.com/hc/en-us/articles/360059641133",
+    "license": "Freeware",
+    "url": "https://download01.logi.com/web/ftp/pub/techsupport/gaming/Onboard%20Memory%20Manager.exe",
+    "hash": "0abe30ac5946bbd5ceab283ddbcc84f553497b9c1019b28f4d1428461343c4d1",
+    "shortcuts": [
+        [
+            "Onboard%20Memory%20Manager.exe",
+            "Onboard Memory Manager"
```

Comment:
```suggestion
            "Onboard Memory Manager.exe",
            "Onboard Memory Manager"
```

Reply:
This doesn't work, it is explicitly named: `Onboard%20Memory%20Manager.exe`
I am fine with adding a bin rename to the `preinstall` though.